vendor/lib/semantic/version.rb

```diff
@@ -10,7 +10,7 @@ class Version
     attr_reader :build
 
     def initialize version_str
-      v = version_str.match(SemVerRegexp)
+      v = version_str&.match(SemVerRegexp)
```

Comment:
Are there no tests for this file?

Reply:
It seems it belongs to a third party gem and doesn't have a test file.